### PR TITLE
[ErrorHandler][HttpKernel] Add missing param annotation for $fileLinkFormat

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | -
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: src/Symfony/Component/ErrorHandler/ErrorRenderer/HtmlErrorRenderer.php [hunk 1]
```diff
@@ -40,8 +41,9 @@ class HtmlErrorRenderer implements ErrorRendererInterface
     private $logger;
 
     /**
-     * @param bool|callable $debug        The debugging mode as a boolean or a callable that should return it
-     * @param bool|callable $outputBuffer The output buffer as a string or a callable that should return it
+     * @param string|FileLinkFormatter|null $fileLinkFormat
+     * @param bool|callable                 $debug          The debugging mode as a boolean or a callable that should return it
```
Comment (derrabus) [inline src/Symfony/Component/ErrorHandler/ErrorRenderer/HtmlErrorRenderer.php]:
```suggestion
     * @param bool|callable                 $debug          The debugging mode as a boolean or a callable that should return it
     * @param string|FileLinkFormatter|null $fileLinkFormat
```

The annotations should have the same order as the actual parameters.

Comment (nicolas-grekas) [reply src/Symfony/Component/ErrorHandler/ErrorRenderer/HtmlErrorRenderer.php]: thanks, updated.

**Review by derrabus — APPROVED** [review]

**Review by yceruto — APPROVED** [review]
Thanks!

**Comment by derrabus** [comment]
Good catch, thanks Nicolas.

**Pull request merged**